packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -57,13 +73,25 @@ const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid, hid
         </div>
         <PatientBannerPatientInfo patient={patient} />
         <div className={styles.buttonCol}>
-          {!hideActionsOverflow ? (
-            <PatientBannerActionsMenu
-              patientUuid={patientUuid}
-              actionsSlotName={'patient-actions-slot'}
-              isDeceased={patient.deceasedBoolean}
-            />
-          ) : null}
+          <div className={styles.buttonRow}>
+            {showPrintIdentifierStickerButton && (
+              <Button
+                kind="ghost"
+                hasIconOnly={true}
```

Comment:
```suggestion
                hasIconOnly
```

Truthy boolean prop values can be omitted.